src/resources/usage/visitusage.txt

```diff
@@ -64,8 +66,8 @@
     Version options
     ---------------------------------------------------------------------------
         -version             Do NOT run VisIt. Just print the current version.
-        -git_version         Do NOT run VisIt. Just print the Git version it
-                             was built from.
+        -git_revision        Do NOT run VisIt. Just print the Git revision
```

Comment:
Just wanna make sure...we are talking about an actual VisIt *version* number (among all the versions appearing on our GitHub releases page) and not a git hash here?